game-summary.py

```diff
@@ -92,14 +93,23 @@
 driver.maximize_window()
 screenshot_obj = driver.save_screenshot("screenshot.png")
 
-# TODO: intelligently crop the screenshot based on the table size
+# Crop the screenshot based on the table size
 # from PIL import Image
+num_games = len(american_league_results)
+
+if len(national_league_results) > num_games:
+    num_games = len(national_league_results)
+
+if len(interleague_results) > num_games:
+    num_games = len(interleague_results)
```

Comment:
```suggestion
num_games = max(len(american_league_results), len(national_league_results), len(interleague_results))

```suggestion
num_games = len(american_league_results)

if len(national_league_results) > num_games:
    num_games = len(national_league_results)

if len(interleague_results) > num_games:
    num_games = len(interleague_results)
```
```